src/redist/targets/GenerateMSIs.targets

```diff
@@ -34,6 +45,11 @@
       <SdkMSBuildExtensionsNuspecFile>$(MSBuildThisFileDirectory)packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.nuspec</SdkMSBuildExtensionsNuspecFile>
       <SdkMSBuildExtensionsNupkgFile>$(ArtifactsNonShippingPackagesDir)VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.$(FullNugetVersion).nupkg</SdkMSBuildExtensionsNupkgFile>
       <SdkMSBuildExtensionsSwrFile>$(ArtifactsNonShippingPackagesDir)VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.swr</SdkMSBuildExtensionsSwrFile>
+
+      <!-- Temp directory for light command layouts -->
+      <LightCommandObjDir>$(ArtifactsObjDir)/LightCommandPackages</LightCommandObjDir>
```

Comment:
Do you think we should move this, and https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/packaging-tools.props#L20-L21 into https://github.com/dotnet/arcade/blob/48646648a006e11c5e0e3696abc7d61f44fc0587/src/Microsoft.DotNet.Build.Tasks.Installers/build/Microsoft.DotNet.Build.Tasks.Installers.props?

Reply:
maybe...IIRC i had trouble importing that file when used outside of an msbuild sdk. msbuild attempts to resolve the import prior to doing the package restore (hence why this file uses the tasks directly).

I may be doing something wrong though. Suggestions are appreciated.